src/lqcontrol.jl

```diff
@@ -308,8 +308,8 @@ only for `min(ts_length, lq.capT)`
 represents `x_t`
 - `u_path::Matrix{Float64}` : A k x T matrix, where the t-th column represents
 `u_t`
-- `w_path::Matrix{Float64}` : A j x T+1 matrix, where the t-th column represents
-`lq.C*w_t`
+- `w_path::Matrix{Float64}` : A n x T+1 matrix, where the t-th column represents
+`lq.C*N(0,1)`
```

Comment:
I thought that looked better because at the beginning we say that `w_t` is a white noise, but then `w_path` - which is supposed to hold the entire history of `w_t`'s - actually holds `C * w_t`, so at that point it seemed better to write `C * N(0,1)`. Anyways, I'm happy with reverting that to the original